handlers/generic_file_upload_ce_attachment_v1/README.md

```diff
@@ -0,0 +1,46 @@
+# Kinetic Request CE Attachment Upload With API
+    This handler is used to copy an attachment from one submission to another.
+
+## Parameters
+    [Error Handling]
+        How to handle error conditions in the handler: raise the error, or return error message
+    [Space Slug]
+        Slug of the Space where the handler should be imported
+    [From Submission Id]
+        Submission Id that contains the attached file to copy.
+    [From Form Field Name]
+        Name of the file attachment field on the Kinetic Request CE form to copy from
+    [Multipart Params]
+        A JSON object that includes the required multi part parameters to send files to your destination with the provided api 
+ ## Results
+[Handler Error Message]
+  Error message if an error was encountered and Error Handling is set to "Error Message".
+
+[Files]
+  Information on the files that were copied.
+
+[Space Slug]
+  The space slug that was used.
+
+## Sample Configuration
```

Comment:
Again, missing the `Attachment Index` and `File Part Parameter` parameters, and all the Info Values.